shared/login/signup/dumb.desktop.js

```diff
@@ -49,12 +51,18 @@ const userEmailShared = {
   emailErrorText: null,
   submitUserEmail: nullFunc,
   waiting: false,
+  usernameChange: (username: string) => console.log(username),
+  emailChange: (email: string) => console.log(email),
 }
 
 const passphraseShared = {
   ...signupShared,
   passphraseError: null,
   checkPassphrase: nullFunc,
+  pass1Update: (pass1: string) => console.log('pass1'),
```

Comment:
🐑  did you mean to do this or did you mean `'pass1:', pass1` ? I'm fine either way